app/views/shopping_carts/show.html.haml

```diff
@@ -0,0 +1,17 @@
+%h2 Your Shopping Cart
+%table.shopping-cart
+  %tr
+    %th Name
+    %th Price
+  - @cart.shopping_cart_items.each do |cart_item|
+    %tr
```

Comment:
Yes we need to address this in the css foundation PT story or maybe later when we are doing the final touches to the design. WE have also mentioned that we need to move the checkout button and cartitem counter to the nav bar when its ready.